### ISSUE TITLE
Mini-Cart: aria-label value contains price even if "Display total price" attribute is set to false

### ISSUE THREAD

**Issue by Aljullu** [woocommerce/woocommerce-gutenberg-products-block#9645]
## To reproduce

1. Add the Mini-Cart block to a post or page and set the _Display total price_ attribute to false.
2. Go to the frontend and with your browser devtools (<kbd>F12</kbd>) instead the `aria-label` value.
3. Notice it says "X items in cart, total price of Y".

## Expected behavior

If _Display total price_ attribute is set to false, I would expect `aria-label` to not contain the price, so it should be something like "X items in cart".
